collections/hub/golden_path/feature_tests/duplicate_handling/transfers/fulfill_commit.json

```diff
@@ -668,179 +698,76 @@
                 ""
               ]
             }
+          }
+        },
+        {
+          "id": 7,
+          "description": "Send Quote - Response",
```

Comment:
How do we know its ok to send the Quote Response here?

I don't see any await inboundEvent listener in the `POST /Quotes` to determine it was received by the NORESPONSE_SIMPAYEE_NAME.

Otherwise we may have a race condition with the `PUT /Quotes` request below.